src/components/RouteWrapper.tsx

```diff
@@ -624,6 +625,14 @@ export const ROUTES: RouteProperties[] = [
         description: "Captures data packets' information and displays in different formats.",
         category: "Network Scanning and Enumeration",
     },
+    {
+        name: "Unicornscan",
+        path: "/tools/Unicornscan",
+        element: <Unicornscan />,
+        description:
+            "Unicornscan is essential for scanning servers and hosts to see what available ports are being utilized for network communications.",
```

Comment:
Changed to Australian spelling - utilised with an "s". 
```suggestion
            "Unicornscan is essential for scanning servers and hosts to see what available ports are being utilised for network communications.",
```